src/puppetlabs/puppetdb/utils.clj

```diff
@@ -339,3 +339,15 @@
   (sp/transform [sp/ALL]
                 #(update % 0 underscores->dashes)
                 m))
+
+(def ^:const one-megabyte (* 1000 1000))
```

Comment:
I'm wondering if this should be base 2 (ie. mebibytes) rather than base 10, since most admins will expect files to be in MiB. I'm not sure yet what the right answer is, just sayin'.